explainer.md

```diff
@@ -542,6 +542,16 @@ There is a related effort to expose motion-sensing controllers through the Gamep
 
+### What's the deal with WebVR?
+
+There's understandably some confusion between the WebXR and an API that some browsers have implemented at various points in the past called WebVR. Both handle communication with Virtual Reality hardware, and both have very similar names. So what's the difference between these two APIs?
+
+**WebVR** was an API developed in the earliest days of the current generation of Virtual Reality hardware/software, started around the time that the Oculus DK2 was announced. Native VR APIs were still in their formative stages, and the capabilities of commercial devices were still being determined, and as such the WebVR API developed around some assumptions that would not hold true long term. For example, the API assumed that applications would always need to render a single left and right eye view of the scene, that the separation between eyes would only ever involve translation and not rotation, and that only one cannonical tracking space was necessary to support. In addition, the API design made forward compatibility with newer device types, like mobile AR, difficult, to the point that it may have necessitated a separate API. WebVR also made some questionable descisions regarding integration with the rest of the web platform, specifically in terms of how it interacted with WebGL and the Gamepad API. Despite this, it worked well enough in the short term that some UAs, especially those shipped specifically for VR devices, decided to ship the API to their users.
+
+In the meantime the group that developed WebVR recognized the issues with the initial API, in part through feedback from developers and standards bodies, and worked towards resolving them. Eventually they recognized that in order to create a more scalable and more ergonomic API they would have to break backwards compatibility with WebVR. This new revision of the API was referred to as WebVR 2.0 for a while, but eventually was officially renamed **WebXR** in recognition of the fact that the new API would support both VR and AR content. Developement of WebXR has been able to benefit not only from the group's experience with WebVR but also from a more mature landscape of immersive computing devices that now includes multiple commercial headsets, the emergence of both mobile and headset AR, several competing and mature native APIs, and the work of other standards bodies addressing the same space.
+
+The intent is that WebXR will completely replace WebVR in the coming years, with all browsers that initially shipped WebVR commiting to shipping WebXR in it's place once the API design is finished. In addition, the WebXR Polyfill provides an implementation of WebXR that can utilize WebVR if the browser expoese it instead, allowing developers to make use of the newer API in all cases.
```

Comment:
> The intent is that WebXR will completely replace WebVR in the coming years, with all browsers that initially shipped WebVR commiting to shipping WebXR in it's place once the API design is finished.

The grammar here was a bit hard to parse... not sure which way is best to fix it..